src/triage/component/catwalk/baselines/thresholders.py

```diff
@@ -7,6 +7,28 @@
 REQUIRED_KEYS = frozenset(["feature_name", "operator", "threshold"])
 
 
+def validate_operator(operator_string, rule):
```

Comment:
I see you are just moving this method; but, just to note, now that you've made it more "public," it is a little misleading that it's not *just* validation – rather, it's *the* way to look up the operator method name.

For example, it could simply be called `valid_method`, `get_valid_method` or even `get_operator_method`.